main/api/helpers.py

```diff
@@ -35,7 +35,7 @@ def handle_error(e):
     }), e.code
 
 
-def make_response(data, marshal_table, next_cursor=None, prev_cursor=None):
+def make_response(data, marshal_table, next_cursor=None):
```

Comment:
I really doubt that with only three params and one optional that would be the case :)